up-and-running.md

```diff
@@ -1,16 +1,17 @@
 # DRAFT
-# Up and running with the Ansible Service Broker
+# Up and running with the Ansible Broker
 
 ## Setup
 In this blog we will discuss how to get up and running with the OpenShift
 Ansible broker. What is the Ansible broker you ask?
 
-> The OpenShift Ansible broker (OAB) is an implementation of the Open Service
-> Broker (OSB) API that manages applications defined by Ansible playbook bundles
+> The OpenShift Ansible Broker (OAB) is an implementation of the Open Service
+> Broker (OSB) API that manages applications defined by Ansible Playbook Bundles
```

Comment:
this was actually a direct quote from the OpenShift documentation :)
https://docs.openshift.org/latest/architecture/service_catalog/ansible_service_broker.html see the first paragraph. That's why they were lower case. But I do like yours since it is more consistent.